frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -2,30 +2,171 @@
-	default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
-	records = [{'doctype': 'Gender', 'gender': d} for d in default_genders]
-	records += [{'doctype': 'Salutation', 'salutation': d} for d in default_salutations]
-	for record in records:
-		doc = frappe.new_doc(record.get("doctype"))
-		doc.update(record)
-
-		try:
-			doc.insert(ignore_permissions=True)
-		except frappe.DuplicateEntryError as e:
-			# pass DuplicateEntryError and continue
-			if e.args and e.args[0]==doc.doctype and e.args[1]==doc.name:
-				# make sure DuplicateEntryError is for the exact same doc and not a related doc
-				pass
-			else:
-				raise
\ No newline at end of file
+    default_genders = [_("Male"), _("Female"), _("Other")]
+    default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
+    records = [{'doctype': 'Gender', 'gender': d} for d in default_genders]
+    records += [{'doctype': 'Salutation', 'salutation': d} for d in default_salutations]
+    for record in records:
+        doc = frappe.new_doc(record.get("doctype"))
+        doc.update(record)
+
+        try:
+            doc.insert(ignore_permissions=True)
+        except frappe.DuplicateEntryError as e:
+            # pass DuplicateEntryError and continue
+            if e.args and e.args[0] == doc.doctype and e.args[1] == doc.name:
+                # make sure DuplicateEntryError is for the exact same doc and not a related doc
+                pass
+            else:
+                raise
+
+
+def make_administrative_area_fixtures():
+
+    administrative_areas = get_administrative_areas()
+    frappe.utils.background_jobs.enqueue(
+        make_administrative_areas,
+        queue="long",
+        timeout=36000,
+        administrative_areas=administrative_areas,
+        enqueue_after_commit=True,
+    )
+
+
+def get_administrative_areas():
+    with open("../apps/frappe/frappe/desk/page/setup_wizard/administrative_areas.json") as f:
+        return json.loads(f.read())
+
+
+def bulid_administrative_area_tree(administrative_areas, country):
+    administrative_states = [area for area in administrative_areas[country] if area['administrative_area_type'] == "state"]
+    lft = 1
+    for state in administrative_states:
+        all_children = [area for area in administrative_areas[country] if area['parent_unique_name'].startswith(state['self_unique_name'])]
+        all_county_children = [area for area in administrative_areas[country] if area['parent_unique_name'] == state['self_unique_name']]
+        rgt = lft + 2*len(all_children) + 1
+        state.update({"lft": lft, "rgt": rgt})
+        if (rgt - lft) == 1:
+            lft = lft + 2
+        else:
+            lft = lft + 1
+            for county in all_county_children:
+                all_children = [area for area in administrative_areas[country] if area['parent_unique_name'].startswith(county['self_unique_name'])]
```

Comment:
shall we delegate this to the NestedSet class? This will be hard to debug! Esp if there are no test cases